fuzzers/050-pip-seed/Makefile

```diff
@@ -3,6 +3,9 @@ ifeq ($(QUICK),Y)
 N ?= 1
 SEGMATCH_FLAGS=
 else
+ifeq (${XRAY_DATABASE}, zynq7)
+N=60
```

Comment:
Should be "N ?= 60" to accommodate "QUICK".

Reply:
Sure, I will fix that right away.

Reply:
Still not fixed?